### DISCUSSION TITLE
git-lfs

### DISCUSSION THREAD

**Discussion by mhanberg** [zk-org/zk#116] [Help] [answered]
Does the repo need the git-lfs features?

I am trying to set up the `--HEAD` functionality for my  [homebrew tap](https://github.com/mhanberg/homebrew-zk) and I realized that you can't use that feature if the repo needs git-lfs.

The `.gitattributes`file is what has the git-lfs configuration I think.

**Comment by mickael-menu** [comment, marked as answer]
I use Git LFS for the binary files (mostly PNG and test database fixtures), but it's not really necessary for `zk` as the resources are quite small. I'm fine removing it from the repo if that makes things easier for you.

**Reply by mhanberg** [reply]
It would make it slightly easier to test out newer features before a release. It's not a huge deal, I just wanted to see if git-lfs was somehow added by accident or wasn't used anymore and could be removed.

If you find it useful, it's fine to keep. 

Thanks!

**Reply by mickael-menu** [reply]
I removed it in `main`. I can't say it was really useful for `zk`, I added it by force of habit. 😄

**Reply by mhanberg** [reply]
`brew install --HEAD zk` is working now, thanks!
